<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>定时器</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>定时器</string>
   </property>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOneShot">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>动态创建单次定时</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>53</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>定时周期</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxIntv">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioContinue">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>95</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>连续定时</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioSingalShot">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>95</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>单次定时</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>461</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>定时器精度</string>
   </property>
   <widget class="QRadioButton" name="radioPrecise">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>PreciseTimer</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioCoarse">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CoarseTimer</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioVeryCoarse">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>VeryCoarseTimer</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>461</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>当前时间（小时：分：秒）</string>
   </property>
   <widget class="QLCDNumber" name="LCDHour">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>12.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="LCDMin">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>41.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="LCDSec">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>27.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelElapsedTime">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
